cabal.project

```diff
@@ -10,8 +10,12 @@ source-repository-package
 source-repository-package
     type: git
     location: https://github.com/fendor/hie-bios.git
-    tag: d5b7fc9bb3025b1d4d2ac9c48b588faf18dfce99
+    tag: 87db34de1b10b03bb2c3d7f6bd3623bc1da96ba8
```

Comment:
actually it got merged, so we could use regular master again

Reply:
what is the correct version to use at present? I am currently using 87db34de1b10b03bb2c3d7f6bd3623bc1da96ba8